IEEE-PSH/hackpsh-web
# Announcements Prototype With Blocknote

# Description
Used Blocknote rich-text editor to allow admins to create announcements. The theme colors should sync up properly. Currently, new announcements are stored in an array and displayed on the same page.
# Test Guidelines
- Go to localhost:3000/make-announcements.
- Check if submitted announcements are displayed as shown in editor.
# Notes
- Plan to fix CSS (padding, etc.).
- Must connect to db.
- Should not conflict with master
